src/assets/sass/variables.scss

```diff
@@ -0,0 +1,23 @@
+$topHeight: 60px;
+$footerHeight: 85px;
+
+$topBgColor: #141515;
+$topGrayColor: #4D5356;
+$bgColor: #0D0E0E;
+$hoverBgColor: #161616;
+$bgMenuColor: #141515;
+$textMenuColor: #9EA1A6;
+$bgMenuTooltip: #0B76DB;
+
+$btnBgColor: #0B76DB;
+
+$reactionYellowColor: #FFC841;
+$reactionOrangeColor: #E25822;
+$reactionRedColor: #F71B26;
+$reactionGreenColor: #00BC75;
+$reactionBlueColor: #0B76DB;
+
+$teamColor: #000;
+$edgeColor: #0B76DB;
+$lifetimeColor: #07B3FF;
+$coachColor: #FAA300;
```

Comment:
When we bring in Stylesora, we can use the tailwind color variables. I've added a lot of the colors from Musora's color palette. But we can always extend the tailwind.config to add more.